lib/Jsrt/Jsrt.cpp

```diff
@@ -4293,11 +4293,28 @@ _ALWAYSINLINE JsErrorCode CompileRun(
 {
     PARAM_NOT_NULL(scriptVal);
     VALIDATE_JSREF(scriptVal);
+    PARAM_NOT_NULL(sourceUrl);
```

Comment:
This needs to be fixed in all other APIs as well. Lets take that as a separate change.